omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -150,6 +178,11 @@
     component_runit_service service do
       action :disable
     end
+  elsif node['private_chef']['opscode-solr4']['external'] &&
```

Comment:
Should we also disable this in any case we have an indexing mode that doesn't use it?

Reply:
which would be elasticsearch only right?

Reply:
I was looking at this attribute 
```
default['private_chef']['opscode-erchef']['search_queue_mode'] = "batch" # rabbitmq, batch, or inline
```
I think only `rabbitmq` mode requires opscode-expander.